### PR TITLE
chore(flame_steering_behaviors): v0.2.0

### PR DESCRIPTION
**Pull request by alestiago**
## Status

**READY**

## Description

Related to #63

Prepares release.

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [X] 🗑️ Chore

### CODE REVIEW

**Review by tomarra — DISMISSED** [review]

**Comment by alestiago** [comment]
I'm drafting this for now. The package flame_steeting_behaviors has a dependency on flame_behaviors so let's get that updated first and then we can bump it before releasing flame_steeting_behaviors.

**Comment by tomarra** [comment]
Before we merge this in we want to attempt a workflow change to see if we can get the automated publishing working.

File: .github/workflows/flame_steering_behaviors_pub_publish.yaml [hunk 1]
```diff
@@ -18,7 +18,15 @@ jobs:
         uses: actions/checkout@v4
       - name: 🐦 Setup Flutter
         uses: subosito/flutter-action@v2
+      - name: 🪪 Get Id Token
+        uses: actions/github-script@v6
+        with:
+          script: |
+            let pub_token = await core.getIDToken('https://pub.dev')
+            core.exportVariable('PUB_TOKEN', pub_token)
```
Comment (tomarra) [inline .github/workflows/flame_steering_behaviors_pub_publish.yaml:26]: At the org level I think this is called `PUB_CREDENTIALS`.

Comment (alestiago) [inline .github/workflows/flame_steering_behaviors_pub_publish.yaml:26]:
```suggestion
            core.exportVariable('PUB_CREDENTIALS', pub_token)
```

Comment (alestiago) [reply .github/workflows/flame_steering_behaviors_pub_publish.yaml:26]: I think this is not trying to read from the organisation secrets. Instead, it is getting the OIDC credentials using `core.getIDToken` (https://github.com/actions/toolkit/blob/main/packages/core/README.md#oidc-token) and then setting it into a variable named `PUB_TOKEN` to be used later on the Authenticate step.

Comment (tomarra) [reply .github/workflows/flame_steering_behaviors_pub_publish.yaml:26]: Ah ok.

**Review by tomarra — APPROVED** [review]

**Review by wolfenrain — APPROVED** [review]

**Pull request merged**